linux/tools.Dockerfile

```diff
@@ -4,7 +4,7 @@
 
 # To build yourself locally, override this location with a local image tag. See README.md for more detail
 
-ARG IMAGE_LOCATION=cdpxb787066ec88f4e20ae65e42a858c42ca00.azurecr.io/official/azure/cloudshell:1.0.20220906.1.base.master.12f76cc7
+ARG IMAGE_LOCATION=cdpxb787066ec88f4e20ae65e42a858c42ca00.azurecr.io/official/azure/cloudshell:1.0.20230109.1.base.master.14ab2c0a
```

Comment:
Please make sure to use this base image tag: azure-cloudshell:1.0.20221130.1.base.master.f5906848

Reply:
updated